scripts/BUILD.bazel

```diff
@@ -247,6 +247,22 @@ kt_jvm_binary(
     ],
 )
 
+kt_jvm_binary(
```

Comment:
### Todo:
Update with removal of test file exemptions

Reply:
removed the data attribute for `compute_changed_files`, will update for retrieve_changed_files as needed as per https://github.com/oppia/oppia-android/pull/5465#discussion_r1707454378

Reply:
Is this addressed now (i.e. can this conversation thread now be resolved)?

Reply:
Yeah, will resolve it as appropriate changes are made.